tests/src/Interop/PInvoke/Generics/GenericsNative.cpp

```diff
@@ -0,0 +1,140 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <stdio.h>
+#include <stdint.h>
+#include <xplatform.h>
+#include <platformdefines.h>
+
+struct Point1F
```

Comment:
I would like to see 64-bit integers as well as `double` types tested as well.

Reply:
Can do.

Reply:
Fixed.